scripts/_0_3_0_percent_matching_probes.py

```diff
@@ -1,6 +1,6 @@
 import utils.db_utils as db_utils
 
-def find_matching_probe_percentage(mongoDB, EVAL_NUMBER=4):
+def main(mongoDB, EVAL_NUMBER=4):
```

Comment:
This will always use an `EVAL_NUMBER` of 4, no?

Reply:
No! It will not. I have a script that is supposed to run weekly (it's called something like weekly ph1 script) that uses this and another script for phase 1, so I converted them to work for 4 and 5